apps/banking_api/lib/banking_api/accounts/deposit.ex

```diff
@@ -0,0 +1,11 @@
+defmodule BankingApi.Accounts.Deposit do
+  @moduledoc """
+  Uses the Operation module to make deposits.
+  """
+  alias BankingApi.Accounts.Operation
+
+  def call(params) do
+    params
+    |> Operation.call(:deposit)
+  end
```

Comment:
```suggestion
  def call(params), do: Operation.call(params, :deposit)
```
Seguindo as sugestões de formatação do credo para [Single pipe](https://hexdocs.pm/credo/Credo.Check.Readability.SinglePipe.html#content).

Deixar a declaração em úma única linha é mais frescura mesmo, mas o single pipe vale a pena.